spec/domain/operations/transfers/to_enroll_batch_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require 'aca_entities/serializers/xml/medicaid/atp'
+require 'aca_entities/atp/transformers/cv/family'
+
+describe Transfers::ToEnrollBatch, "given a soap envelope with an valid xml payload, transfer it to Enroll" do
+  include Dry::Monads[:result, :do]
+
+  let(:transfer_id) { "tr123" }
+  let(:p1) { File.read("./spec/test_data/xmls/MET00000000002007409_larry.xml") }
+  let(:p2) { File.read("./spec/test_data/xmls/MET00000000002007409_larry_suzie.xml") }
+  let(:p3) { File.read("./spec/test_data/xmls/MET00000000002007409_larry_cheryl.xml") }
+  let!(:transfer1) { FactoryBot.create(:inbound_transfer, external_id: transfer_id, payload: p1, to_enroll: true, result: "Waiting to Transfer")}
+  let(:process) { described_class.new }
+
+  context 'no matching transfers' do
+    before :each do
+      @result = process.call("randomstring")
+    end
+
+    it 'should return success message' do
+      expect(@result).to_not be_success
```

Comment:
The description of this example looks like it should be 'should return failure message'?  Also, I don't think this is failing in the expected way, which is likely due to the issue in the above comment. 
`Failure("get_payload error: undefined method `update!' for nil:NilClass")`
Could check for the expected Failure message:
`expect(@result.failure).to eq("no transfers found")`